packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -164,7 +164,38 @@ Media.createMediaConnection = (
       {
         iceServers,
       },
-      debugId
+      debugId,
+      (messages, context) => {
+        // Convert argument `messages` object to an array.
+        const logMessages = Array.from(messages).map((message) =>
+          typeof message === 'object' ? JSON.stringify(message) : message
+        );
+
+        logMessages.unshift(`[${context.name}]`);
```

Comment:
it would be useful to prepend also the debugId, so like this:
```
logMessages.unshift(`${debugId}:[${context.name}]`);
```
because normally debugId is included in all logs coming from internal-media-core

Reply:
updated in `internal-media-core`